cabal-install/Distribution/Client/CmdInstall.hs

```diff
@@ -131,14 +131,20 @@ import System.FilePath
 data NewInstallFlags = NewInstallFlags
   { ninstInstallLibs :: Flag Bool
   , ninstEnvironmentPath :: Flag FilePath
+  , ninstForceOverwrite :: Flag Bool
   }
 
 defaultNewInstallFlags :: NewInstallFlags
 defaultNewInstallFlags = NewInstallFlags
   { ninstInstallLibs = toFlag False
   , ninstEnvironmentPath = mempty
+  , ninstForceOverwrite = toFlag False
   }
 
+boolToOverwritePolicy :: Bool -> OverwritePolicy
+boolToOverwritePolicy True  = DoOverwrite
+boolToOverwritePolicy False = DontOverwrite
```

Comment:
Still trying to figure out how to just have a `Flag OverwritePolicy` without having to reimplement trueArg